pages/vitalik.js

```diff
@@ -317,7 +287,7 @@ export default function Vitalik() {
                   Hack Club
                 </Link>{' '}
                 is a global community of high school makers & student-led coding
-                clubs. We’ve got a 24/7 Slack chatroom of 15k+ teenagers
```

Comment:
I think it should be 15k+

<img width="127" alt="Screen Shot 2022-02-04 at 11 28 15 AM" src="https://user-images.githubusercontent.com/34525547/152565723-6b79a449-2ded-4a9c-88dd-1494fad6300a.png">

Reply:
> re squashing: it's best to start a new branch for every change and base it off hackclub:main
> […](#)
> On Sat, 5 Feb 2022, 00:28 Caleb Denio, ***@***.***> wrote: ***@***.**** commented on this pull request. ------------------------------ In pages/vitalik.js <[#339 (comment)](https://github.com/hackclub/site/pull/339#discussion_r799622702)>: > @@ -317,7 +287,7 @@ export default function Vitalik() { Hack Club </Link>{' '} is a global community of high school makers & student-led coding - clubs. We’ve got a 24/7 Slack chatroom of 15k+ teenagers I think it should be 15k+ [image: Screen Shot 2022-02-04 at 11 28 15 AM] <https://user-images.githubusercontent.com/34525547/152565723-6b79a449-2ded-4a9c-88dd-1494fad6300a.png> — Reply to this email directly, view it on GitHub <[#339 (review)](https://github.com/hackclub/site/pull/339#pullrequestreview-873310273)>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/AJP3VRGBUQZGQPZVKH3XRGDUZP5DVANCNFSM5NSHFA5A> . You are receiving this because you were mentioned.Message ID: ***@***.***>

thanks sam!